DigitalLearningSolutions.Web/Helpers/ConfigHelper.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace DigitalLearningSolutions.Web.Helpers
 {
     using System;
+    using System.Drawing;

```

Comment:
Is this needed?

Reply:
I keep getting these caused by resharper thinking it knows better, in this case it was Icon instead of IConfiguration i think, and the cleanup code doesn't reliably remove unused usings.